weld_rt/cpp/Makefile

```diff
@@ -1,5 +1,5 @@
 OS = $(shell uname -s)
-LLVM_VERSION = 3.8
+LLVM_VERSION = $(shell llvm-config --version | cut -d . -f 1,2)
```

Comment:
Hmm this shouldn't change behavior if 3.8 is the installed version as the script as it is already appends `-3.8`. This just allows it to also work with other versions, so long as `llvm-config` is on the `PATH`. On my MacOS, installing via `brew install llvm` creates  `/usr/local/opt/llvm/bin/clang-4.0`.